src/CalcManager/Ratpack/exp.cpp

```diff
@@ -345,8 +345,14 @@ void powratNumeratorDenominator(PRAT *px, PRAT y, uint32_t radix, int32_t precis
         // ##################################
         // Round the originalResult to roundedResult
         // ##################################
+        PRAT rat_digits = nullptr;
+        DUPRAT(rat_digits, rat_ten);
+        // Round to 32 digits after decimal point
+        powratcomp(&rat_digits, longtorat(32), radix, precision);
```

Comment:
Please assign the call to `longtorat` to a variable and call `destroyrat()` on the variable after use to avoid a memory leak.